lib/src/util.dart

```diff
@@ -131,3 +131,55 @@ String escapePunctuation(String input) {
 
   return buffer.toString();
 }
+
+extension StringExtensions on String {
+  /// Removes leading whitespace by the length of [length].
```

Comment:
Maybe "Removes up to [length] characters of leading whitespace."

Reply:
Fixed, thanks!